src/api/app/models/bs_request.rb

```diff
@@ -107,6 +108,21 @@ def check_creator
     errors.add(:creator, "Login #{user.login} is not an active user")
   end
 
+  def double_actions
```

Comment:
the "header" isn't a thing in ruby classes. so i would also prefer to use the scopes as it breaks the code into smaller chunks which makes it easier to understand and to test.